### ISSUE TITLE
 can't send long commands with <CR> + <LF>

### ISSUE THREAD

**Issue by oumartraore** [eelcocramer/node-bluetooth-serial-port#183]
Hi all!
First, thanks for this module it is really good that you shared it. 

I use ascii protocole in some project  to send some command like 'rb01'. But it not working, i think your librarie send buffer caractere by caractere.
How to make it possible to send all command, like we use putty or tera-term with <CR>+<LF> setting ?

**Comment by eelcocramer** [comment]
Multiple bytes can be send within a single packet but there is no guarantee that the complete buffer is send in a single packet.

**Comment by eelcocramer** [comment]
That said, if you are sure that a command would fit with the MTU of a packet it should be possible to send transfer the command in a whole. I'm not sure if the current code will guarantee this though, I would need to check the code for each platform to be sure.

**Comment by oumartraore** [comment]
How to specify the size of my buffer, to allow the sending of all my buffer in a single packet ? I am on windows. Even if it's in code c

**Comment by eelcocramer** [comment]
This is where the buffer is written on windows: https://github.com/eelcocramer/node-bluetooth-serial-port/blob/master/src/windows/BTSerialPortBinding.cc#L98

As for the other question: https://nodejs.org/api/events.html#events_emitter_removelistener_eventname_listener

**Comment by oumartraore** [comment]
I thank you for your answer. I found the solution to my problem.

As you said, multiple bytes can be send, but there is no guarantee that the entire buffer is sent in a packet.

I used another library for the "serial port" able to send the data in "string" or in a "buffer".

I think that if we communicate through a protocol, it is more judicious to send a "string" than a "buffer".

this could help another newbie. I thank you for helping me
